modules/server/src/graphqlRoutes.js

```diff
@@ -146,12 +146,20 @@ const createEndpoint = async ({
     return next();
   });
 
+  console.log(' \n');
+
   return router;
 };
 
-export default async ({ enableAdmin, esClient, getServerSideFilter, graphqlOptions = {} }) => {
+export default async ({
+  configsSource = '',
```

Comment:
sometimes I wish our prettier setup left these in multiple lines so this doesn't happen.